sdk/storage/azblob/internal/exported/blob_batch.go

```diff
@@ -0,0 +1,271 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package exported
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/log"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/uuid"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/internal/shared"
+	"io"
+	"mime"
+	"mime/multipart"
+	"net/http"
+	"net/textproto"
+	"strconv"
+	"strings"
+)
+
+type BlobBatchOperationType string
```

Comment:
Since this isn't actually exported, but is required by exported content, it would be good to add a comment explaining that.  In fact. if there's other content in here that's not exported, it would be best to group them all together at the end of the file ([example](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/internal/exported/request.go#L173-L182)).

Reply:
Done